packages/babel-helper-skip-transparent-expression-wrappers/src/index.js

```diff
@@ -0,0 +1,37 @@
+import * as t from "@babel/types";
+import type { NodePath } from "@babel/traverse";
+
+export function getCallContext(callPath: NodePath): NodePath {
```

Comment:
I think `getCallContext` is a consumer of `helper-skip-transparent-expression-wrappers` and it should not be in this package.

Do we have a test case on `Expected type "CallExpression" or "OptionalCallExpression" `? I think we can go with `skipTransparentExprWrappers(path.get("callee"))` and not worry about `path.type`, they should be validated by transformers or implied by how the `path` is filtered. i.e. `"CallExpression|OptionalCallExpression"(path)`.

Reply:
I'm alright with that. I think it made more sense when the scope of the package was smaller.